models/google_ads__account_report.sql

```diff
@@ -25,9 +25,12 @@ fields as (
         accounts.time_zone,
         sum(stats.spend) as spend,
         sum(stats.clicks) as clicks,
-        sum(stats.impressions) as impressions
+        sum(stats.impressions) as impressions,
+        sum(stats.conversions) as conversions,
+        sum(stats.conversions_value) as conversions_value,
+        sum(stats.view_through_conversions) as view_through_conversions
 
-        {{ fivetran_utils.persist_pass_through_columns(pass_through_variable='google_ads__account_stats_passthrough_metrics', transform = 'sum') }}
+        {{ google_ads_persist_pass_through_columns(pass_through_variable='google_ads__account_stats_passthrough_metrics', identifier='stats', transform='sum', coalesce_with=0, exclude_fields=['conversions','conversions_value','view_through_conversions']) }}
```

Comment:
I don't believe this is necessarily a blocker, but the approach for adding the default conversion fields is a bit different between Google Ads and LinkedIn Ads. For Google Ads we are adding the fields in explicitly; however, in LinkedIn Ads we are adding them in dynamically through the use of the conversions variable that has default values. In LinkedIn this allows the user to add the aliased conversion field to their passthrough column variable and it will still work and update the name of the field in the end model to be the aliased name. However, for Google Ads we explicitly define the conversion fields and therefore the alias the customer may provide is completely overwritten.

For example, if I use the following variable config:
```yml
vars:
  google_ads__account_stats_passthrough_metrics:
    - name: conversions
      alias: net_new_conversions
```

I only see the field `conversions` in the end model, but not `net_new_conversions`. In the LinkedIn Ads PR it is actually the other way around. This may/may not add complexity when trying to roll up the conversions in the ad report, but the LinkedIn Ads `total_conversions` field is likely the catch all that we would use and therefore isn't a real concern.

That being said, this isn't something that needs to be addressed in the code per se; however, this is a fairly large breaking change as it will revert any aliases for these fields that customers add and therefore could also cause the downstream ad reporting passthrough to break. I would either like us to explore updating the README/CHANGELOG of this package and the Ad Reporting package to address this, OR update the code to ensure if an aliased field is provided, it will still be passed through in addition to the required field name.

Reply:
Huh when I run the package with that variable config, I do indeed see `net_new_conversions`
![image](https://github.com/user-attachments/assets/da839033-4545-4e81-830d-d3407cf55f20)

These [lines](https://github.com/fivetran/dbt_google_ads/blob/feature/convesion_metrics/macros/google_ads_persist_pass_through_columns.sql#L7-L8) of the macro are intended to handle aliases

Reply:
Thanks for the details here @fivetran-jamie! This was a blunder on my part as I was querying the wrong table when validating this part of the PR 🤦. I was able to rerun using the correct variable:table combination and can see the expected results. No change required here. Thanks!